message PBMsgReqGds{  optional string type = 1;  optional int32 worldId = 2;  optional int32 version = 3;}